src/Components/InventoryListRow/InventoryListRow.js

```diff
@@ -0,0 +1,76 @@
+import './InventoryListRow.scss'
+import ChevronIcon from '../../Assets/Icons/chevron_right-24px.svg'
+import BinIcon from '../../Assets/Icons/delete_outline-24px.svg'
+import PencilIcon from '../../Assets/Icons/edit-24px.svg'
+import { Link } from 'react-router-dom'
+import React, {useEffect, useRef} from 'react'
+
+
+const InventoryListRow = (props) => {  
+
+    const statusTextElement = useRef([])
+
+    useEffect(()=>{
+        if (props.status === 'In Stock'){
+            statusTextElement.current.classList.add('inventoryListRow__status--active')
+        } else{
+            statusTextElement.current.classList.add('inventoryListRow__status--inactive')
+        }
+    })
+
+    return(
+        <>
+            <div className='inventoryListRow'>
+                <div className='inventoryListRow__container inventoryListRow__container--primary'>
+                    <div className='inventoryListRow__header-container'>
+                        <p className='inventoryListRow__header'>INVENTORY ITEM</p>
+                    </div>
+                    <div className='inventoryListRow__subContainer'>
+                        <Link className='inventoryListRow__link' to={`${props.itemID}`}>
```

Comment:
```suggestion
                        <Link className='inventoryListRow__link' to={`/inventory/${props.itemID}`}>
```
It appears the route doesn't automatically add the `/inventory/` part